scripts/multi_thermostat_driver.py

```diff
@@ -39,20 +39,24 @@
 # logging.json: Normal logging example
 # logging_noisy.json: Turns on all debugging information
 # logging_quiet.json: Only logs error messages
-LOGGING_CONFIG = 'logging.json'
+LOGGING_CONFIG = 'scripts/logging.json'
 
 # Save cached weather data files? (Default: False)
 SAVE_CACHE = False
 CACHE_PATH = None  # Replace with location to save the weather cache files (e.g. '/tmp/epathermosat')
+TAU_SAVE_PATH = Path('outputs')
 
 # This section finds the metadata files and data files for the thermostats.
 # These point to examples of the various styles of files
 # In most cases you will combine Single Stage, Two Stage, and Two Stage ERT
 # data in the same file.
 
 # Single Stage
-DATA_DIR = Path('../../datadir/EPA_Tau')
-METADATA_FILENAME = DATA_DIR / '2019_epa_tau.csv'
+HVAC_THERMOSTAT = "AC_Const"
```

Comment:
Swtich between AC_const and AC_comb for constant and combined (Night+Setback) thermostat algorithms